scss/colors.scss

```diff
@@ -8,25 +8,29 @@ $twitter-blue: #1da1f2;
   --background-color: white;
   --text-color: #{$human-blue};
   --post-btn-background-color: #{$human-blue};
+  --post-btn-background-color-hover: #153F72;
```

Comment:
I can't see any difference in the color when I hover over the light or dark theme buttons.